tensorflow/python/ops/nn_ops.py

```diff
@@ -1836,7 +1836,7 @@ def softmax_cross_entropy_with_logits_v2(
     name: A name for the operation (optional).
 
   Returns:
-    A `Tensor` of the same shape as `labels` and of the same type as `logits`
+    A `Tensor` of shape `[batch_size]` and of the same type as `logits`
```

Comment:
Maybe also change the docstring for the softmax_cross_entropy_with_logits() function below this one.